go/vt/topo/helpers/tee.go

```diff
@@ -188,7 +190,7 @@ func (ld *teeTopoLockDescriptor) Unlock(ctx context.Context) error {
 
 	if serr != nil {
 		if ferr != nil {
-			log.Warningf("First Unlock(%v) failed: %v", ld.dirPath, ferr)
+			log.Warningf("First Unlock(%v, %v) failed: %v", ld.cell, ld.dirPath, ferr)
```

Comment:
Note that you've got inconsistency in error messages: in some places cell and path are separated just by space, in some they are separated by comma. I'd personally prefer the latter format (with the comma), but in any case it should be consistent across all the messages.